flax/experimental/nnx/README.md

```diff
@@ -61,9 +60,61 @@ assert model.count == 1
 
 In this example `nnx.Rngs(0)` create a `random.key` for `params` with seed `0`, this is used by `rngs.<rng-name>()` inside `__init__` to generate a random key to initialize the parameters.
 
-### Training with the Functional API
+### Interacting with JAX
+
+While NNX Modules inherently follow reference semantics, they can be easily converted into a pure functional representation that can be used with JAX transformations and other value-based, functional code.
+
+NNX has two very simple APIs to interact with JAX: `split` and `merge`.
```

Comment:
I'm wondering if you can add a small comment on how `split` and `merge` differs from `bind` and `unbind` in Linen and what the advantages are, which could give the users more incentive to try out NNX